JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -228,8 +228,8 @@ public function denormalize($data, $class, $format = null, array $context = [])
         );
 
         foreach ($normalizedData as $attributeName => $attributeValue) {
-            // Ignore JSON-LD special attributes
-            if ('@' === $attributeName[0]) {
+            // Ignore JSON-LD special attributes and null attribute names
+            if ('@' === $attributeName[0] || is_null($attributeName[0])) {
```

Comment:
There still needs to be a way to determine if the attribute name is null.  The error is still thrown using 
```
if (0 === strpos($attributeName, '@') {
```